.azuredevops/modulePipelines/ms.synapse.workspaces.yml

```diff
@@ -0,0 +1,66 @@
+name: 'Synapse - Workspaces'
+
+parameters:
+  - name: removeDeployment
+    displayName: Remove deployed module
+    type: boolean
+    default: true
+  - name: versioningOption
+    displayName: The mode to handle the version increments [major|minor|patch]
+    type: string
+    default: patch
+    values:
+      - patch
+      - minor
+      - major
+  - name: customVersion
+    displayName: Custom version to apply. Used only if higher than latest
+    type: string
+    default: '0.0.1'
+
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - '/.azuredevops/modulePipelines/ms.synapse.workspaces.yml'
+      - '/.azuredevops/pipelineTemplates/module.*.yml'
+      - '/arm/Microsoft.Synapse/workspaces/*'
+    exclude:
+      - '/**/*.md'
+
+variables:
+  - template: '/.azuredevops/pipelineVariables/global.variables.yml'
+  - group: 'PLATFORM_VARIABLES'
+  - name: modulePath
+    value: '/arm/Microsoft.Synapse/workspaces'
+
+stages:
+  - stage: Validation
+    displayName: Pester tests
+    jobs:
+      - template: /.azuredevops/pipelineTemplates/module.jobs.validate.yml
+
+  - stage: Deployment
+    displayName: Deployment tests
+    jobs:
+      - template: /.azuredevops/pipelineTemplates/module.jobs.deploy.yml
```

Comment:
The templates have been renamed in `main` (yesterday). Please rebase to latest main and update the references